x-pack/plugins/apm/server/lib/traces/get_trace.ts

```diff
@@ -37,10 +57,46 @@ export async function getTrace(
             }
           ]
         }
+      },
+      aggs: {
+        transactions: {
+          terms: {
+            field: 'transaction.id'
+          },
+          aggs: {
+            errors: {
+              filter: {
+                term: {
+                  'processor.event': 'error'
+                }
+              }
+            }
+          }
+        }
       }
     }
   };
 
-  const resp = await client<Span | Transaction>('search', params);
-  return resp.hits.hits.map(hit => hit._source);
+  const resp = await client<Span | Transaction, TraceAggregationResponse>(
+    'search',
+    params
+  );
+  const trace = resp.hits.hits.map(hit => hit._source);
+  const transactionErrorCounts = resp.aggregations.transactions.buckets.reduce(
+    (
+      acc: object,
+      { key, errors: { doc_count } }: TraceErrorsAggregationBucket
+    ) => {
```

Comment:
Maybe it's me, but I found this line a bit hard to parse. What do you think about:

```js
(acc, bucket: TraceErrorsAggregationBucket) => {
```
?

Reply:
oh whoops. i meant to update the name before committing